plugins/org.komodo.eclipse.workspace.ui/src/org/komodo/eclipse/workspace/ui/editor/panels/KObjectEditorPanel.java

```diff
@@ -0,0 +1,129 @@
+package org.komodo.eclipse.workspace.ui.editor.panels;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.ModifyEvent;
+import org.eclipse.swt.events.ModifyListener;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+import org.komodo.spi.repository.KomodoObject;
+
+public class KObjectEditorPanel  extends Composite {
+	private KomodoObject docNode;
+	private String type;
+
+	private Text nodeTypeText;
+	
+//	private IDocChangedListener listener;
+	
+	private boolean synching = false;
+	
```

Comment:
Are all the commented out portions of code necessary in the future. My concern with them is that such code is 'sticky' and never seems to be erased from the codebase and as such better to not add it initially.